analyses/fusion_filtering/README.md

```diff
@@ -0,0 +1,47 @@
+## Fusion prioritization
+
+**Module author:** Krutika Gaonkar ([@kgaonkar6](https://github.com/kgaonkar6))
+Jaclyn Taroni ([@jaclyn-taroni] (https://github.com/jaclyn-taroni))
+Jo Lynne([@jharenza](https://github.com/jharenza))
+Komal Rathi ([@komalsrathi] (https://github.com/komalsrathi))
+
+
+This analysis will perform artifact filtering and annotation on fusion calls from STARfusion and Arriba fusion callers to prioritize putative oncogenic fusions. 
+Briefly, we considered all inframe and frameshift fusion calls with a minimum of 1 junction reads and at least one gene partner expressed (TPM > 1) to be true calls. 
+If a fusion call had large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten), we removed these calls as potential false positives. 
+We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers OR the same fusion was recurrent within a broad_histology (>2 samples) OR the fusion was specific to the broad_histology. 
```

Comment:
What does prioritized mean here? Does it mean did not remove or something else? It's possible that I don't understand the filtering steps/starting point instead of not understanding this.

Reply:
The starting point for this step is QC filtered fusion calls (readthrough, annotated by FusionAnnotator with red-flag/normal fusion 
 database  and expression based filtering)
Now in this step we want to do project specific filtering where the aim is to **keep** fusions that fit the conditions : detected by both callers (which would be our high confidence calls) OR the same fusion was recurrent within a broad_histology (>2 samples to find recurrent fusions per histology) OR the fusion was specific to the broad_histology (that is only seen in this specific histology ) as FilteredFusion.tsv output file.